Dialog/RoomConfigDialog.cpp

```diff
@@ -234,6 +257,11 @@ void RoomConfigDialog::on_ComboBox_TilesetID_currentIndexChanged(int index)
             ui->ComboBox_Layer0Picker->addItem(
                 QString::number(WL4Constants::ToxicLandfillDustyLayer0Ptr, 16).toUpper());
         }
+        else if (ui->ComboBox_TilesetID->currentIndex() == 0x45)
```

Comment:
Let's make this a constant (and if it is used somewhere else, also change that usage)

Reply:
Done weirdly.